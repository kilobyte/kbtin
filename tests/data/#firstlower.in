#foreach {abc Abc ąbc Ąbc AOw Aow aOw aOw aów aÓw Ąów ĄÓw} {#firstlo res {$0};#show $res}
