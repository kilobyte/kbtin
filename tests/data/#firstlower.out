abc
abc
ąbc
ąbc
aOw
aow
aOw
aOw
aów
aÓw
ąów
ąÓw
